commands_function.py

```diff
@@ -0,0 +1,59 @@
+import os
+from abc import ABCMeta, abstractstaticmethod
+import json
+from keys import keys
+
+class Icommand(metaclass=ABCMeta):
+    @abstractstaticmethod
+    def excute():
+        """the interface Icommand"""
+
+class commandfactory:
```

Comment:
file name should be `command_factory` not `commands_factory`

Reply:
ok done